src/sentry/integrations/github/integration.py

```diff
@@ -28,12 +30,31 @@
     aspects={}
 )
 
+API_ERRORS = {
+    404: 'GitHub returned a 404 Not Found error.',
```

Comment:
@mnoble open to rewording, but this is consistent with our old plugin: https://github.com/getsentry/sentry-plugins/blob/master/src/sentry_plugins/github/plugin.py#L26 so i'd rather we just file a ticket and revisit the messaging later.

Also, I think the main reason we had it phrased this way was to make it clear that the error comes from the GitHub API, but there is probably a clearer way we could do that.

@MeredithAnya maybe you could add something similar to what the old plugin has like "If this repository exists, ensure that you have authorized Sentry to access it." since I'd guess this would happen if people didn't give us access to a particular repo, but then tried to add it anyway?

Reply:
@macqueen @mnoble going to file a JIRA ticket to get back to this.